tests/test_worker.py

```diff
@@ -449,3 +451,26 @@ class Foo:
     assert await j1.status() == JobStatus.complete
 
     assert await arq_redis.enqueue_job('foobar', _job_id='job_id') is None
+
+
+async def test_custom_serializers(arq_redis_msgpack: ArqRedis, worker):
+    j = await arq_redis_msgpack.enqueue_job('foobar', _job_id='job_id')
+    worker: Worker = worker(functions=[foobar], job_serializer=msgpack.packb, job_deserializer=functools.partial(msgpack.unpackb, raw=False))
+    await worker.main()
+    assert await j.result() == 42
+    r = await j.info()
+    assert r.result == 42
+
+
+async def test_incompatible_serializers_1(arq_redis_msgpack: ArqRedis, worker):
+    await arq_redis_msgpack.enqueue_job('foobar', _job_id='job_id')
+    worker: Worker = worker(functions=[foobar])
+    with pytest.raises(SerializationError) as exc_info:
```

Comment:
what's the error here? Might be good to add `assert str(exc_info.value) == ...` just to document what the error is.

And below.